src/components/ReportWelcomeText.tsx

```diff
@@ -1,53 +1,52 @@
 import React, {useMemo} from 'react';
```

Comment:
**Archived Workspace Invoice Chat**

| Staging | PR |
| ------------- | ------------- |
| <img width="1054" alt="archived-ws-invoice-chat-staging" src="https://github.com/user-attachments/assets/93536da4-107c-412f-8a16-9102c5bdff94"> | <img width="1133" alt="archived-ws-invoice-chat" src="https://github.com/user-attachments/assets/9ff60ad5-5218-4cf1-8473-94173bee62e3"> |

This is a Staging vs PR before / after of an archived workspace invoice chat, the way you get an archived workspace invoice chat is by:
1. Create new WS what is able to send invoices.
2. FAB > Send Invoice > Select WS and Send.
3. Settings > Workspaces > Delete WS.
4. Notice the archived workspace invoice chat welcome text.

Observe the discrepancies between Staging and PR screenshots, missing **(archived)** and the addition of WS name.